### ISSUE TITLE
On initial install impersonations icons shown but not possible because not configured

### ISSUE THREAD

**Issue by sharidas** [owncloud/impersonate#136]
From the UI perspective it would help users to understand who can be impersonated. Instead of showing icons listed who may/may not be impersonated.

**Comment by tomneedham** [comment]
Blocker to a release IMO because since #134 the app is installed with a unusable configuration and none of the icons work with no explanation.

**Comment by tomneedham** [comment]
Ah maybe this is the case.... I didn't know because I couldn't see the error. I thought there was no defualt configuration given the new change.

**Comment by PVince81** [comment]
ref https://github.com/owncloud/impersonate/issues/17

**Comment by sharidas** [comment]
I got a small idea. Inorder to hide the users who have not logged in at least once:

- We get the data of each users in a row.
   1. When the data of a user is arrived, it does have `lastlogin` ( or `lastLogin`, I am not able to recollect it now ), `<td>`. May be we can check what is the status and allow to show that.

But there are other cases, like nested impersonation ( which is obviously out of scope of this ticket ). In such cases we might have to think how to handle it by analysing response from the server. Meaning changes would have to be made in the server so that client analyses the response and shows/does not show impersonate icon.

**Comment by felixboehm** [comment]
@tomneedham what timeline for release is needed?
This p1 and needs a hotfix? Or can we set p2 for "must have" in next release?

**Comment by PVince81** [comment]
after discussing we clarified that the issue already existed and was accepted as "by design for now" with https://github.com/owncloud/impersonate/issues/17

so I think this should be put down to p3 until the concern for https://github.com/owncloud/impersonate/issues/17 becomes bigger

@tomneedham can you confirm ?

**Comment by tomneedham** [comment]
Correct. Sorry, I misused p1 here as "must have next release". And yes, clarified, can be p3 - no decrese in functionality.
